guides/common/modules/proc_preparing-for-using-external-databases.adoc

```diff
@@ -0,0 +1,27 @@
+[id="Preparing_for_using_external_databases_{context}"]
+= Optional: Preparing {Project} for using external databases
+
+By default, the {Project} installation process includes installing a PostgreSQL database on the same host as {ProjectServer}.
+However, in certain {Project} deployments, using external databases instead of the default local databases can help with the server load.
+
+You can set up your {Project} deployment to use an external PostgreSQL database during the installation of {Project}. 
+You can also migrate to an external PostgreSQL database on an existing {Project} deployment.
+
```

Comment:
What would you think about dropping these two lines? Line 7 doesn't add anything new -- that information is already expressed through the section heading, the intro, and the steps themselves. And line 8 is not relevant here because at this point in the process, users are preparing for the installation; these users don't need to know what to do when they already have an existing installation. Moreover, line 8 duplicates line 27.

Dropping these lines would help keep the installation prep content shorter, which is one of the goals of the content journey.